src/emu/epoc/src/services/window/classes/winbase.cpp

```diff
@@ -259,6 +259,18 @@ namespace eka2l1::epoc {
             return true;
         }
 
+        case EWsWinOpOrdinalPosition: {
```

Comment:
These two opcodes in Symbian source code send opcode using WriteReply command.
When these are called, reply are written to request status

so it should be

```cpp
        case EWsWinOpOrdinalPosition: {
            ctx.set_request_status(ordinal_position(false));
            break;
```

Same with lower